src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -27,113 +203,1113 @@ const ActorInit En_Test3_InitVars = {
+            rot->z += this->player.unk_AAC.x;
+        } else if (limbIndex == OBJECT_TEST3_LIMB_0A) {
+            s32 pad; // Must exist here to match
+
+            if (this->player.unk_AA8 != 0) {
+                Matrix_RotateZS(0x44C, MTXMODE_APPLY);
+                Matrix_RotateYS(this->player.unk_AA8, MTXMODE_APPLY);
+            }
+            if (this->player.unk_AB2.y != 0) {
+                Matrix_RotateYS(this->player.unk_AB2.y, MTXMODE_APPLY);
+            }
+            Matrix_RotateXS(this->player.unk_AB2.x, MTXMODE_APPLY);
+            if (this->player.unk_AB2.z != 0) {
+                Matrix_RotateZS(this->player.unk_AB2.z, MTXMODE_APPLY);
+            }
+        } else {
+            func_80125500(play, &this->player, limbIndex, pos, rot);
+        }
+    }
+    return false;
+}
+
+void EnTest3_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList1, Gfx** dList2, Vec3s* rot, Actor* thisx) {
+    s32 pad;
+    EnTest3* this = THIS;
+
+    if (*dList2 != NULL) {
+        Matrix_MultZero(D_80A41D6C);
+    }
+    if (limbIndex == OBJECT_TEST3_LIMB_10) {
+        MtxF curMtxF;
+        Actor* leftHandActor;
+
+        Math_Vec3f_Copy(&this->player.leftHandWorld.pos, D_80A41D6C);
+        if (*dList1 != NULL) {
+            func_80128640(play, &this->player, *dList1);
+            if (this->player.stateFlags3 & PLAYER_STATE3_20000000) {
+                OPEN_DISPS(play->state.gfxCtx);
+                gSPDisplayList(POLY_OPA_DISP++, object_test3_DL_00EDD0);
+                CLOSE_DISPS(play->state.gfxCtx);
+            }
+        }
+        leftHandActor = this->player.heldActor;
+        if ((leftHandActor != NULL) && (this->player.stateFlags1 & PLAYER_STATE1_800)) {
+            Vec3s curRot;
+
+            Matrix_Get(&curMtxF);
+            Matrix_MtxFToYXZRot(&curMtxF, &curRot, false);
+            leftHandActor->world.rot.y = this->player.actor.shape.rot.y + this->player.leftHandWorld.rot.y;
+            leftHandActor->shape.rot.y = leftHandActor->world.rot.y;
+        } else {
+            Matrix_Get(&this->player.mf_CC4);
+            Matrix_MtxFToYXZRot(&this->player.mf_CC4, &this->player.leftHandWorld.rot, false);
+            func_80126B8C(play, &this->player);
+        }
+
+    } else if (limbIndex == OBJECT_TEST3_LIMB_13) {
+        Actor* leftHandActor = this->player.heldActor;
+
+        if (leftHandActor != NULL) {
+            leftHandActor->world.pos.x = 0.5f * (this->player.bodyPartsPos[15].x + this->player.leftHandWorld.pos.x);
+            leftHandActor->world.pos.y = 0.5f * (this->player.bodyPartsPos[15].y + this->player.leftHandWorld.pos.y);
+            leftHandActor->world.pos.z = 0.5f * (this->player.bodyPartsPos[15].z + this->player.leftHandWorld.pos.z);
+        }
+    } else if (limbIndex == OBJECT_TEST3_LIMB_0B) {
+        Actor* actor730 = this->player.unk_730;
+
+        if ((*dList1 != NULL) && this->player.currentMask && !(this->player.stateFlags2 & PLAYER_STATE2_1000000)) {
```

Comment:
```suggestion
        if ((*dList1 != NULL) && (this->player.currentMask != PLAYER_MASK_NONE) && !(this->player.stateFlags2 & PLAYER_STATE2_1000000)) {
```

Reply:
Doesn't match. I'll leave a comment about it. Also thinking the conditional might be more clear with the negation out front.

Reply:
Decided against that.